egs/wsj/s5/steps/nnet3/nnet3_train_lib.py

```diff
@@ -351,7 +475,7 @@ def ComputePresoftmaxPriorScale(dir, alidir, num_jobs, run_opts,
     WriteKaldiMatrix(output_file, [scaled_counts])
     ForceSymlink("../presoftmax_prior_scale.vec", "{0}/configs/presoftmax_prior_scale.vec".format(dir))
 
-def PrepareInitialAcousticModel(dir, alidir, run_opts):
+def PrepareInitialAcousticModel(dir, alidir, run_opts, use_raw_nnet = False):
```

Comment:
If you just want to initialize a raw nnet which is not an acoustic model, you should just create a wrapper for the `nnet3-init` and not try to reuse this method.